leaf-ui/js/object/InputAnalysis.js

```diff
@@ -12,7 +12,7 @@ class InputAnalysis {
 		this.initialAssignedEpoch = savedAnalysisData.finalAssignedEpoch ? savedAnalysisData.finalAssignedEpoch : ['0'];
 		this.initialValueTimePoints = savedAnalysisData.finalValueTimePoints ? savedAnalysisData.finalValueTimePoints:
 			['0'];
-		this.elementList = getElementsForAnalysis();
+		//this.elementList = getElementsForAnalysis();
```

Comment:
Is the InputAnalysis class even used anymore? I know it mirrors an InputAnalysis class in the backend, but I couldn't find it being used anywhere in the frontend. Wondering if we even need to have it